samcli/commands/check/bottle_necks.py

```diff
@@ -0,0 +1,73 @@
+"""
+Bottle neck questions are asked here. Data is saved in graph, but not calcualted here.
+"""
+
+import click
+
+
+class BottleNecks:
+    def __init__(self, graph):
+        self.graph = graph
```

Comment:
Does `self.graph` need to be public member?
If not, let's make it protected by changing its name to `self._graph`

One underscore > protected
Two underscore > private